downloader/history_downloader/__init__.py

```diff
@@ -0,0 +1,12 @@
+import ccxt
+
+class HistoryDownloader:
+    exchange = ccxt.binance() #creating instance of binance exchange
+    def __init__(self,crypto_symbol:str)->None:
+        self.crypto_symbol = crypto_symbol
+
+    def get_history(self):
+        """ downloading cryptocurrency history """
+        return self.exchange.fetch_ticker(self.crypto_symbol)
+
+print(type(HistoryDownloader('BTC/USDT').get_history()))
```

Comment:
In formal PRs please do not include your `work in progress` code. Make sure that everything is working fine, but always remove debugging code.

Reply:
Yes, sorry about that. I just forgot to clean it up.